README.md

```diff
@@ -7,32 +7,37 @@ A GitHub Action that will run [Flyway](https://flywaydb.org/) against a specifie
+| `db-name`                     | true        | N/A     | The name of the database to run flyway against.                                                                                                                                                                                            |
+| `migration-files-path`        | true        | N/A     | The path to the base directory containing the migration files to have flyway process.                                                                                                                                                      |
+| `flyway-command`              | true        | N/A     | The flyway command to run; e.g `migrate`, `validate`, etc.                                                                                                                                                                                 |
+| `migration-history-table`     | true        | N/A     | The table where the migration history lives. This is most likely dbo.MigrationHistory or Flyway.MigrationHistory.                                                                                                                          |
+| `baseline-version`            | false       | 0       | The baseline version to send to the flyway command.                                                                                                                                                                                        |
+| `managed-schemas`             | true        | N/A     | A comma separated list of schemas that are to be managed by Flyway.MigrationHistory.                                                                                                                                                       |
+| `enable-out-of-order`         | false       | false   | A switch that allows new migrations that are a lower version number than a current migration to be run.                                                                                                                                    |
+| `validate-migrations`         | false       | true    | A switch determining whether flyway should validate the migration scripts before running them.                                                                                                                                             |
+| `use-integrated-security`     | false       | false   | A switch defining whether or not to use integrated security. If not provided, a password should be.                                                                                                                                        |
+| `use-azure-managed-identity`  | false       | false   | A switch that can be used to indicate that an Azure Managed Identity should be used to authenticate with the SQL Server.                                                                                                                   |
```

Comment:
Should the description indicate you need to include the azure-msi-client-id arg if you use this?

Reply:
Good point. I did the vice-versa of that, but it should probably go here too.